lib/membrane_raw_audio.ex

```diff
@@ -40,21 +40,20 @@ defmodule Membrane.RawAudio do
   @type t :: %Membrane.RawAudio{
           channels: channels_t,
           sample_rate: sample_rate_t,
-          format: Format.t()
+          sample_format: SampleFormat.t()
         }
 
-  defstruct channels: nil,
-            sample_rate: nil,
-            format: nil
+  @enforce_keys [:channels, :sample_rate, :sample_format]
+  defstruct @enforce_keys
 
   @doc """
   Returns how many bytes are needed to store a single sample.
 
   Inlined by the compiler
   """
   @spec sample_size(t) :: integer
-  def sample_size(%__MODULE__{format: format}) do
-    {_, size, _} = Format.to_tuple(format)
+  def sample_size(%__MODULE__{sample_format: format}) do
+    {_, size, _} = SampleFormat.to_tuple(format)
```

Comment:
This repo has an outdated credo config. Please use one from the current plugin template and fix found issues.
One will be reported here:
```suggestion
    {_type, size, _endian} = SampleFormat.to_tuple(format)
```